src/runtime-rs/crates/runtimes/virt_container/src/lib.rs

```diff
@@ -149,4 +161,29 @@ agent_name="kata"
         let res = new_agent(&toml_config);
         assert!(res.is_ok());
     }
+
+    #[test]
+    fn test_new_hypervisor() {
+        VirtContainer::init().unwrap();
+
+        let toml_config = {
+            let config_content = r#"
+[hypervisor.qemu]
+path = "/bin/echo"
+kernel = "/bin/echo"
+image = "/bin/echo"
+firmware = ""
+
+[runtime]
+hypervisor_name="qemu"
+            "#;
+            TomlConfig::load(config_content).map_err(|e| anyhow!("can not load config toml: {}", e))
+        }
+        .unwrap();
+
+        let res = tokio::runtime::Runtime::new()
```

Comment:
Maybe using `#[tokio::test]` instead of `#[test]` can avoid manually calling `tokio::runtime`.

Reply:
Good point, fixed, thanks!